core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -773,6 +773,10 @@ private[spark] class SparkSubmit extends Logging {
           childArgs += ("--arg", arg)
         }
       }
+      // Pass the proxyUser to the k8s app so it is possible to add it to the driver args
```

Comment:
So, this is adding backing `--proxy-user` in order to propagate it to K8s?

Reply:
Yes!